storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -592,24 +577,13 @@
       (doseq [p (set/difference (set (keys existing-assignment))
                                 (set (keys new-assignment)))]
         (.killedWorker isupervisor (int p)))
```

Comment:
`StandaloneSupervisor` implementation does nothing. What is the need of this API?